src/chapters/orm-intro/background.rst

```diff
@@ -0,0 +1,170 @@
+Object-Relational Mapping
+=========================
+
+.. index:: ! Object-Relational Mapping, ! ORM, ! Data Layer, ! object-relational mapper
+
+With all that we have learned about MVC applications and relational databases, we are ready to connect the two and add persistent data storage to our apps!
```

Comment:
Sally introduced me to the idea of running my text through http://www.hemingwayapp.com/ to try and reach a target of ~5th grade reading level. its going to be very difficult to reach that with these concepts and vocab words in the mix but something closer to that is an improvement. now that im using it myself, i see a big difference in writing style between this first page and the earlier material

Reply:
Nice. Two of the pages here register at grade 4 and 5. The background page is a grade 6, and while I wasn't able to get it down to 5, I did restructure and improve several sentences.